ng/src/web/components/form/textarea.js

```diff
@@ -22,20 +23,41 @@
  */
 
 import React from 'react';
-
-import {classes} from 'gmp/utils';
+import glamorous from 'glamorous';
 
 import compose from '../../utils/compose.js';
 import PropTypes from '../../utils/proptypes.js';
+import Theme from '../../utils/theme.js';
 
 import withLayout from '../layout/withLayout.js';
 
 import withChangeHandler from './withChangeHandler.js';
 
+const StyledTextArea = glamorous.textarea({
+  /* use font and line settings from parents not from browser default */
+  fontFamily: 'inherit',
+  fontSize: 'inherit',
+  lineHeight: 'inherit',
+
+  display: 'block',
+  height: 'auto',
+  color: Theme.darkGray,
+  backgroundColor: Theme.white,
+  backgroundImage: 'none',
+  border: '1px solid' + Theme.inputBorderGray,
+  borderRadius: '4px',
+  padding: '1px 8px',
+}, ({disabled}) =>
+    disabled ? {cursor: 'not-allowed'} : null,
+    ({disabled, readonly}) => readonly || disabled ? {
+      backgroundColor: Theme.dialogGray,
+      opacity: 1,
+    } : null,
+  );
+
 const TextAreaComponent = ({className, value = '', ...props}) => {
```

Comment:
IMHO the TextAreadComponent is unnecessary now. You could remove it and use StyledTextAread instead.